bdk-ffi/src/types.rs

```diff
@@ -26,12 +26,8 @@ pub struct CanonicalTx {
     pub chain_position: ChainPosition,
 }
 
-impl From<BdkCanonicalTx<'_, Arc<bdk::bitcoin::Transaction>, ConfirmationTimeHeightAnchor>>
-    for CanonicalTx
-{
-    fn from(
-        tx: BdkCanonicalTx<'_, Arc<bdk::bitcoin::Transaction>, ConfirmationTimeHeightAnchor>,
-    ) -> Self {
+impl From<BdkCanonicalTx<'_, Arc<BdkTransaction>, ConfirmationTimeHeightAnchor>> for CanonicalTx {
```

Comment:
thanks for doing things like this in this PR so we can have things like `Arc<BdkTransaction>` instead of `Arc<bdk::bitcoin::Transaction>`, makes it clean and I need to make sure I'm paying attention to this sort of stuff to for any of my new PR's